src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -636,6 +636,14 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
     return this.wasReplaced(this.currentOrgPerson) ? '' : 'ACTIVE'
   }
 
+  /** Flag to show confirm name change checkbox.
+  This check is only for firm corrections, change, conversion, Corp extension and Corp conversion filings.
+  Does not apply to corps corrections */
+  get showConfirmNameChange () : boolean {
```

Comment:
I know naming things is hard, but this is only one condition for identifying that the name has changed. (The other condition is on line 765.)

I can't think of a better name, but I think the code would look OK if you made this logic a variable inside hasNameChanged() and then used that variable on line 756.